src/drc/tests/test_notifications_send.py

```diff
@@ -82,6 +72,7 @@ def test_send_notif_create_enkelvoudiginformatieobject(self, mock_client):
             }
         )
 
+    @skip('HTTP DELETE is currently not supported')
```

Comment:
See earlier remark :-)

Although I don't think this should generate a notification, since ZRC/BRC will already send one out.